### ISSUE TITLE
Improving Brave's Ad-Free mode

### ISSUE THREAD

**Issue by dimitri-xyz** [brave/ledger#4]
## The Trade-Offs

In my view, the immediate trade-off between the two proposed business models can be described as:
- ad-replacement model: does not require users to pay, but users still have to see ads.
- ad-free model: requires payment from users, but provides an (almost) ad-free experience.

Neither model requires (as currently proposed) significant infrastructure change from publishers.

There is however, another difference between the two models that I believe needs to be considered carefully:
- ad-replacement model: Brave is ultimately funded by advertisers, not users.
- ad-free model: Brave is ultimately funded by users, not advertisers.

This is important because over the long run, misaligned incentives can cause Brave to lose touch with its privacy mission, independent of its initial goals. For example, consider the erosion of privacy that happened at Google since its inception. From this point of view, the ad-free business model is more appealing.
## Current problems with Ad-Free

From a user's point of view, I see two value propositions in using Brave's ad-free mode:
1. Avoids visual pollution - User does not see ads and page loads faster
2. Avoids tracking - User privacy is respected

This is the value proposition I believe Brave wants to provide. However, it is not currently possible for a user to determine whether she is getting this. In other words, the current design of ad-free mode does
not provide accountability. This is because:

a. Brave does not block first-party ads, so some ads may still show up.
b. The publisher has _not_ agreed to respect user privacy and may choose to violate it.
### Avoiding visual pollution

Let us first consider (a).

If a user goes to a website and sees _zero_ ads using Brave, the user can immediately see the value of Brave. On the other hand, if the same user goes to a website and sees _fewer_ ads. It is almost impossible for the user to perceive any benefit in using Brave as the user has no baseline comparison level. So, fewer ads just seems like: "this adblocker is not working properly, I can still see some ads." In other words, in ad-free as currently proposed _Users do not know what ad-blocking value they are getting._

In ad-free mode, the user will only perceive a clear benefit in using Brave if no ads at all are seen and, thus, even _first-party_ ads need to be removed. Clearly, this makes the problem harder, but assume for
now that this is possible. There is still the other problem with ad-free: How much should users pay?

How much should I, as a user, be willing to pay to see ad-free content? What is a fair price? Users have no easy way to know this. Users do not know how much they should pay and as a consequence do not know whether they can afford an ad-free web experience. Even if users browsing in ad-free mode saw value in the experience, they may still think that they are over paying for it.

Here's a personal anecdote as an example:
I have been using blendle.com recently. In that website, every article comes with a different price. I find it very hard to justify paying $0.49 for most articles of the Wall Street Journal when I see that most articles from the New York Times are available for $0.19. Why the price difference? Is the Wall Street journal over charging me? (Thankfully, in Blendle I can withhold payment if the article is not good.)

I believe Brave can help users determine how much they should pay for an ad-free web visit. The current ad auctions run by 3rd-parties provide an upper-bound on the price a user should be asked to pay. Today those auctions are run based on user tracking data. But it may be possible to leverage this information to help users and automate "ad-free biding" in the browser. That's my suggestion. If users knew that an ad-free web experience costs approximately $9 a month (or whatever), they may be more willing to fork out the cash. Paying $9 a month without knowing how many ad-free days one is going to get is not a deal I would take.
### Avoiding Tracking

Let us now consider (b).

Currently, publishers are not required to preserve user privacy and they have financial incentives to subvert it. Given the chance, they may come up with ingenious ways to track users. To demonstrate Brave's value proposition, Brave should require that publishers adopt an appropriate "Do Not Track policy" (e.g. https://www.eff.org/dnt-policy) before they are able to receive funds from ad-free users. This forces publishers to be accountable. It doesn't solve the problem, but now Brave users have set a benchmark and can sue publishers who violate their privacy, whereas before there was no such single friendly benchmark and, thus, it was not so easy to sue.
## Conclusion: Ad-Free requires infrastructure changes

If one accepts the premise that users will not pay unless they see value, the above argument has an important consequence: Making the value of ad-free mode visible to users requires cooperation from publishers to establish a good "Do Not Track policy" and to negotiate a fair price for blocking all ads, including first-party ads. Thus, this mode may be a harder sell in the short-term despite keeping Brave on the side of users over the long-term.

As currently proposed, "ad-free mode" will not provide users with incentives to spend money as they will not see any clear value in it. I believe this can and should be addressed.

**Comment by bridiver** [comment]
We have not made any official statements about the cost of ad-free that I am aware of. Are you seeing a statement somewhere that there will be a per-article (or similar) cost?

**Comment by BrendanEich** [comment]
Hi Dmitri, I was part of the Mozilla group who, with Chris Soghoian and Jonathan Mayer, developed DNT (I said the header name needed to be three letters ;-). It failed and will fail again if users have to trust pubs and (hah!) regulators. We know this now from bitter experience.

I may have time to reply at length later today, but I wanted to point out that we are not able to count on most users paying for web content. There is zero evidence that users at scale will do so. Even if some do as we hope (and as Blendle and other efforts confirm), most do not. Many at global scale cannot afford to pay much or anything. So we're focusing not on ad replacement first, rather on a hybrid model. We are not out to insist on pay-to-block -- that is a loser in view of free browsers and free ad blockers for them.

**Comment by dimitri-xyz** [comment]
Hi Everyone,

@bridiver - I didn't see any statement about ad-free costs. Was that somehow implied in my comments? My main argument is for letting users users know exactly what they are paying for.

@BrendanEich - 

I wanted to make two points: 
1. I don't know the whole story, but I believe there was never any financial incentive for publishers to adopt DNT. In fact, they had incentives to lobby regulators not to enforce it. That may be a significant difference between ad-free Brave and DNT. In other words, ad-free Brave could require compliance to a privacy friendly policy to allow publishers to get their money (or simply block ads otherwise).
2. I do not believe that _all_ users will pay for web content. We are in agreement here. But as you noted, some users might and a freemium business model could work (like it does for Evernote). So, I look at a hybrid ad-free/ad-replacement model with good eyes. What I am worried about is that I think _no one_ would pay for ad-free mode _as currently proposed_, because users do not know what they get for their money. "Fewer ads" is not a good sales pitch, "zero ads" is much clearer.

Hope this helps!

**Comment by bridiver** [comment]
@dimitri-xyz imo the biggest problem with DNT is that there is no good way to verify compliance. Even if the pubs we worked with agreed and we had some practical way to verify their continuing compliance (a very big "if"), there would still be plenty of other companies out there trying to find ways around our tracking protection. We are trying to design a system that requires as little user trust as possible, and that includes not having to trust Brave. We know there are tracking methods out there that no one is currently blocking (unique etags, cached scripts, etc...) and we will continue to improve and address those.

**Comment by bridiver** [comment]
I agree that a fixed monthly price is preferable to a per-article payment. Unlike services that require pre-arranged publisher contracts, Brave must be able to collect and escrow payments for pubs that we currently have no relationship with. Since we can't reasonably expect everyone to implement some kind of pricing protocol, there isn't really a practical way to implement publisher controlled per-article pricing anyway, and a fixed per-article price doesn't make sense because not all content has the same value. However, a fixed subscription may or may not be a comprehensive solution and some pubs may prefer to operate as "premium channels". In either case, with the possible exception of paywall pubs, you would never run out of content, you would just be bumped back to ad supported mode.

In terms of "fewer ads" vs "zero ads", "zero ads" is not a realistic goal, but "fewer ads" is really "almost zero ads". The cost of trying to recognize every possible ad is too high compared to the incremental value. Most ads are easily ignored because most pubs use common ad servers and the traffic originates from recognizable domains. It's also possible to look at things like the position and size of an image to determine the likelihood that it is an ad, but 100% detection with an acceptably low false positive rate would be a difficult task to say the least

**Comment by bridiver** [comment]
also just a side note: while most of the media attention has been on the ad supported model, that is not necessarily representative of where our efforts have been focused

**Comment by dimitri-xyz** [comment]
@bridiver  I need to think a little more about your comments. I wasn't really arguing for having a fixed monthly price, I was arguing for letting users know if they can afford an ad-free experience, but a fixed monthly price may be a consequence.

About the "fewer ads" vs "zero ads", I agree that the incremental cost of recognizing every possible ad is very high. However, I am not sure that this incremental cost is higher than the incremental value. I don't think we know that. I think that the incremental value of finding _every single ad_ is also high, so the jury is still out on whether it is worth doing.

One suggestion I have for making the cost of finding ads lower is to crowdsource it. Rather than trying to find all ads by ourselves, Brave could enable its users to find ads and share those findings with others.
